luisy/decorators.py

```diff
@@ -258,4 +256,29 @@ def output_decorator(cls):
             'catalog': catalog,
         }
         return cls
+
     return output_decorator
+
+
+def azure_blob_storage_output(blob=None):
+    def decorator(cls):
+        cls.target_cls = AzureBlobStorageTarget
+        cls.target_kwargs = {
+            'blob': blob,
+        }
+        return cls
+
+    return decorator
+
+
+def azure_blob_storage_input(blob=None):
```

Comment:
Should we explicitly state and forward all constructor arguments of the AzureBlobStorageTarget?